src/config.js

```diff
@@ -414,6 +414,7 @@ class Config {
       this.schema.properties.core.properties.projectHome = projectHomeSchema
       this.defaultSettings.core.projectHome = projectHomeSchema.default
     }
+    this.indicator = []
```

Comment:
What is up with this property? It only seems to store "magic numbers" (1, 2, and 3), but the value is never read, and the array is never cleared, so this will keep accumulating values. Is this debugging code you have simply forgotten to remove?